api/api/spec/spec.yaml

```diff
@@ -2750,6 +2780,82 @@ paths:
         '401':
           description: 'Access unauthorized'
 
+  /lists:
+    get:
+      tags:
+      - lists
+      summary: 'Get all CDB lists'
+      description: 'Returns the contents of all CDB lists. Optionally, the result can be filtered by several criteria. See available parameters for more details.'
+      operationId: api.controllers.lists_controller.get_lists
+      parameters:
+        - $ref: '#/components/parameters/pretty'
+        - $ref: '#/components/parameters/wait_for_complete'
+        - $ref: '#/components/parameters/sort'
+        - $ref: '#/components/parameters/search'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/statusListParam'
+        - $ref: '#/components/parameters/pathFilter'
+      responses:
+        '200':
+          description: 'Successfully got CDB lists'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/ApiResponse'
+                - type: object
+                  properties:
+                    data:
+                      allOf:
+                        - $ref: '#/components/schemas/ListMetadata'
+                        - $ref: '#/components/schemas/CDBList'
+                example:
+                  error: 0
+                  data:
+                    totalItems: 1
+                    items:
+                      -
+                        - key: audit-wazuh-w
+                          value: write
+                        - key: audit-wazuh-r
+                          value: read
+
+  /lists/files:
+    get:
+      tags:
+      - lists
+      summary: 'Get paths from all lists'
+      description: 'Returns the path from all lists. Use this method to know all the CDB lists and their location in the filesystem relative to Wazuh installation folder'
+      responses:
```

Comment:
Missing `operationId` and `parameters` in this endpoint.

Reply:
You are totally right! Plase, find it already fixed.